Makefile

```diff
@@ -288,6 +288,36 @@ personal-dev-env:
 endif
 .PHONY: personal-dev-env
 
+#
+# Local Cluster Service Development Environment
+#
+ifeq ($(DEPLOY_ENV),pers)
+local-pers-dev-env: personal-dev-env
+	@echo ""
+	@echo "===================================================================="
+	@echo "Personal dev environment setup complete"
+	@echo "===================================================================="
+	@echo ""
+	@echo "Granting local development permissions..."
+	@$(MAKE) -C dev-infrastructure local-cs-permissions
+	@echo ""
+	@echo "===================================================================="
+	@echo "Local CS permissions granted successfully"
+	@echo "===================================================================="
+	@echo "Generating local provision-shard config..."
+	@echo ""
+	@cd cluster-service && $(MAKE) local-deploy-provision-shard
+	@echo ""
+	@echo "===================================================================="
+	@echo "Provision shard configuration generated at:"
+	@echo "cluster-service/local/local-provisioning-shards.yml"
+	@echo "===================================================================="
+else
+local-pers-dev-env:
```

Comment:
is this missing https://github.com/Azure/ARO-HCP/blob/deef2f7dda6c8f0c7df78a0aac27129f676743ab/cluster-service/Makefile#L152 ?